output/cloud/expv2/integration/integration_test.go

```diff
@@ -0,0 +1,155 @@
+package integration
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/klauspost/compress/snappy"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.k6.io/k6/cloudapi"
+	"go.k6.io/k6/lib/testutils"
+	"go.k6.io/k6/lib/types"
+	"go.k6.io/k6/metrics"
+	"go.k6.io/k6/output/cloud/expv2"
+	"go.k6.io/k6/output/cloud/expv2/pbcloud"
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/proto"
+	"gopkg.in/guregu/null.v3"
+)
+
+// This test runs an integration tests for the Output cloud.
+// It only calls public API of the Output and
+// it implements a concrete http endpoint where to get
+// the protobuf flush requests.
+
+func TestOutputFlush(t *testing.T) {
+	// TODO: it has 3s for aggregation time
+	// then it means it will execute for +3s that it is a waste of time
+	// because it isn't really required.
+	// Reduce the aggregation time (to 1s?) then run always.
+	t.Skip("Skip the integration test, if required enable it manually")
```

Comment:
You can do the same as we do for tc39 and do not run this on `go test -short`
https://github.com/grafana/k6/blob/ecca7894b60f69b730d88e9f64c57b144171a3f1/js/tc39/tc39_test.go#L689-L691

Reply:
I would like to have it always enabled so moving it to the end of the chain, added it to the reminders list.